waffle-compiler/src/config/inputToConfig.ts

```diff
@@ -48,7 +48,8 @@ export function inputToConfig(input: InputConfig) {
 function validate(config: any): asserts config is Config {
   const success = (
     checkSourceDirectory(config.sourceDirectory) &&
-    checkOutputDirectory(config.outputDirectory) &&
+    checkCompileOutputDirectory(config.compileOutputDirectory) &&
+    checkFlattenOutputDirectory(config.flattenOutputDirectory) &&
     checkNodeModulesDirectory(config.nodeModulesDirectory) &&
     checkCompilerType(config.compilerType) &&
     checkCompilerVersion(config.compilerVersion) &&
```

Comment:
Not related to the task at hand but looks like a good place to use [@restless/sanitizers](https://www.npmjs.com/package/@restless/sanitizers).